packages/wp-story-editor/src/index.js

```diff
@@ -35,7 +35,7 @@ import domReady from '@wordpress/dom-ready';
 /**
  * External dependencies
  */
-import { StoryEditor } from '@googleforcreators/story-editor';
+import { StoryEditor, FontCheck } from '@googleforcreators/story-editor';
```

Comment:
Can `FontCheck` completely live in `wp-story-editor`?

Reply:
Think so... I started it there but was hitting a bunch of lint errors re: importing I'll circle back on that.

Reply:
No rush, we can revisit at the end. Probably some functions would need to be exported first to make it work.

The current way works 👍

Reply:
> Probably some functions would need to be exported first to make it work.

Here's the commit where I added the exports --- is this okay?

https://github.com/GoogleForCreators/web-stories-wp/pull/10808/commits/3fcb74fc1e2d96b7fc7206b2e66d24b97bb8dfa5